proposals/drafts/ocdid_governance.rst

```diff
@@ -0,0 +1,140 @@
+* Such a mechanism may be considered adopted if upon proposal it receives a majority vote from the country's ``committers``.
+* No such mechanism may be added for the top level OCDID hierarchy (``ocd-division/country:``) or for countries who would otherwise not have any identifiers.
+
+If a conversation around a request cannot reach consensus, after two months the person who made the pull request may request a final vote from the country's ``committers``.
+
+Approval will be handled on a per-file, rather than a per-commit, basis.
+
+New commits should include well-formed explanations, especially if generating new IDs or types.
+
+Formalized guidelines for approval will be created for use by ``committers``. These guidelines will include:
+
+* Syntax check guidelines
+* Type check guidelines and a glossary of existing types
+* Dupe check guidelines
+* Instructions for including and checking for correct sameAs aliasing
+* Differentiating standards of review between generative/corrective requests
+
+Government contributions
+~~~~~~~~~~~~~~~~~~~~~~~~
+Should a government entity wish to contribute to the repository, they will initially be asked to work directly with an existing ``committer`` to prepare to integrate their identifier set.
+
+* A second ``committer`` is still required for approving government contributions, even if a government contributor becomes a ``committer`` themselves.
+* Even so, government contributors should be given wide deference within their geographic area.
+* If, because of naming conventions, geographic edge cases, etc. a government contributor requests a deviation from existing OCDID nomenclature, the community will attempt to reasonably accommodate that request (e.g. using “police_jury” as a type in lieu of “county_council” for Louisiana's county legislative body)
+
+Government entities (and the ``committers`` they work with) will be expected to reconcile and appropriately alias these cases with existing OCDIDs in order to ensure maximum compatibility.
+
+Identifiers created by government officials that are used in official data will be marked as such in the repository, so that developers can quickly identify the preferred identifier in case of conflict. Caution should be used, and the original submitter consulted with if possible, before changing government submitted identifiers.
+
+A section of documentation specifically aimed at government staff will be created, where they can learn more about the project and how to get involved, as well as how to reach out to the community to get help.
+
+Support
+--------
+``Committers`` will be expected to participate in a quarterly review of their country's new OCDIDs in order to ensure quality on-going.
+``Committers`` will be requested to contribute and maintain an ongoing style guide for creating new district types.
+``Committers`` will be required to participate in > 60% of their country's formal votes/actions as announced.
```

Comment:
60% may be a high bar. I'm not sure what 'actions' refers to, but if approving PRs is an action, then it may be hard to get there, if other committers are faster than you are.